DMX.Core.Api.Tests.Unit/Services/Orchestrations/LabCommands/LabCommandOrchestrationServiceTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Linq.Expressions;
+using DMX.Core.Api.Brokers.Loggings;
+using DMX.Core.Api.Models.Foundations.LabCommands;
+using DMX.Core.Api.Models.Foundations.LabCommands.Exceptions;
+using DMX.Core.Api.Services.Foundations.LabCommands;
+using DMX.Core.Api.Services.Orchestrations.LabCommands;
+using Moq;
+using Tynamix.ObjectFiller;
+using Xeptions;
+using Xunit;
+
+namespace DMX.Core.Api.Tests.Unit.Services.Orchestrations.LabCommands
+{
+    public partial class LabCommandOrchestrationServiceTests
+    {
+        private readonly Mock<ILabCommandService> labCommandServiceMock;
+        private readonly Mock<ILoggingBroker> loggingBrokerMock;
+        private ILabCommandOrchestrationService labCommandOrchestrationService;
+
+        public LabCommandOrchestrationServiceTests()
+        {
+            this.labCommandServiceMock = new Mock<ILabCommandService>();
+            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+
+            this.labCommandOrchestrationService = new LabCommandOrchestrationService(
+                labCommandService: this.labCommandServiceMock.Object,
+                loggingBroker: this.loggingBrokerMock.Object);
+        }
+
+        public static TheoryData<Xeption> LabCommandDependencyValidationExceptions()
+        {
+            string randomErrorMessage = GetRandomString();

```

Comment:
Either name this `randomMessage` or change the one below - I vote `randomMessage`